Zend/zend_portability.h

```diff
@@ -435,7 +435,7 @@ char *alloca();
 #ifdef ZEND_WIN32
 #define ZEND_SECURE_ZERO(var, size) RtlSecureZeroMemory((var), (size))
 #else
-#define ZEND_SECURE_ZERO(var, size) memset((var), 0, (size))
+#define ZEND_SECURE_ZERO(var, size) explicit_bzero((var), (size))
```

Comment:
Roger, well then it seems good to go to me for master, @nikic any comments?

I can take care of eventual Windows failures once merged, should there be any we did not cover